### PR TITLE
Story/414

### PR DESCRIPTION
**Pull request by ws-mps**

### CODE REVIEW

File: models/stock_move_line.py [hunk 1]
```diff
@@ -426,3 +424,43 @@ def get_info(self):
...
+        mls = self
+
+        if state is None:
+            pass
+        elif state == 'done':
+            mls = mls.filtered(lambda ml: ml.qty_done == ml.product_qty)
+        elif state == 'not_done':
+            mls = mls.filtered(lambda ml: ml.qty_done < ml.product_qty)
+        else:
+            assert False, 'State not valid to sort tasks'
```
Comment (nocturnalastro) [inline models/stock_move_line.py]: shouldn't this be raise error() instead of assert?

File: models/stock_production_lot.py [hunk 2]
```diff
@@ -60,3 +60,26 @@ def get_lot(self, lot_identifier, product_id,
                 % str(lot_identifier))
 
         return results
+
+    def _prepare_info(self):
+        """
+            Prepares the following info of the quant in self:
+            - id: int
+            - name: {stock.quant.package}
```
Comment (nocturnalastro) [inline models/stock_production_lot.py]: shouldn't this be stock.production.lot?

Comment (ws-mps) [reply models/stock_production_lot.py]: yes, copy & paste :)

File: models/stock_move_line.py (hunk 1 above)
Comment (ws-mps) [reply models/stock_move_line.py]: Semantically they are different. If we get here is a coding error, not a system error that is why we have used an assert.

Comment (nocturnalastro) [reply models/stock_move_line.py]: Yes but as assertions can be turned off it wouldn't be safer to raise an AssertionError

**Pull request merged**